docs/api/proposed_endpoints.md

```diff
@@ -17,13 +17,13 @@ first_name  | text      | first name of the user
 last_name   | text      | last name of the user
 email_id    | text      | email of user. Errors out if email already registered.
 password    | text      | Hash of the user's password
-gender      | choices   | gender (Like Male, Female, Others)
-tshirt_size | choices   | size of the TShirt (Like Small, Medium etc.)
+gender      | text      | gender (Like Male, Female, Others)
```

Comment:
the choices should be in backquotes, all lower case like `male`, `female` etc.

Please make this change everywhere else.